src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx

```diff
@@ -106,3 +123,18 @@ function getDropMessage(multiple: boolean) {
       return `Choose or drop ${multiple ? 'files' : 'a file'} here.`
   }
 }
+
+function getTooLargeMessage() {
+  switch (currentLanguage()) {
+    case 'de':
+      return `Eine oder mehrere Dateien sind zu groß. Bitte Dateien bis maximal ${prettyBytes(
```

Comment:
* [ ] FYI "Bitte laden Sie..." sounds more friendly.
* [ ] UX: It would be nice to mention the limit *before* users start adding/dropping files.

Reply:
> UX: It would be nice to mention the limit before users start adding/dropping files.

I want to keep the text brief for uploading and only show the information when needed. It will also be shown right away without an upload delay.